src/createjs/utils/BrowserDetect.js

```diff
@@ -63,7 +63,7 @@ this.createjs = this.createjs||{};
 	 */
 	function BrowserDetect() {
 		throw "BrowserDetect cannot be instantiated";
-	};
```

Comment:
We don't usually add semi-colons on function references. The prototype functions are different because they are declarations of the value on the prototype. Recommend removing this change (and any others in this PR)